features/formatter/use_the_tap_formatter.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Use the TAP formatter
+  So that I can get non-XML parseable results
+  As a Developer
+  I need to be able to use a TAP formatter
+
+  Scenario: Empty spec causes various result states
+    Given the spec file "spec/Formatter/TapExample1/TapSpec.php" contains:
+      """
+      <?php
+
+      namespace spec\Formatter\TapExample1;
+
+      use PhpSpec\ObjectBehavior;
+      use PhpSpec\Exception\Example\SkippingException;
+
+      class TapSpec extends ObjectBehavior
+      {
+          function it_is_most_definitely_pending()
+          {
+          }
+
+          function it_is_most_definitely_passing()
+          {
+              $this->fire('pass')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_failing()
+          {
+              $this->fire('fail')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_broken()
+          {
+              $this->fire('broken')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_skipping()
+          {
+              throw new SkippingException('php is not installed');
+          }
+      }
+      """
+    And the class file "src/Formatter/TapExample1/Tap.php" contains:
+      """
+      <?php
+
+      namespace Formatter\TapExample1;
+
+      use PhpSpec\Exception\Example\ErrorException;
+
+      class Tap
+      {
+          public function fire($stuff)
+          {
+              switch ($stuff) {
+                  case 'pass':
+                      return 'pass';
+                  case 'fail':
+                      return 'fail';
+                  case 'broken':
+                      throw new ErrorException('error','something terrible occurred','foo.php',99);
+              }
+          }
+      }
+      """
+    When I run phpspec using the "tap" format
+    Then I should see:
+      """
+      TAP version 13
+      not ok 1 - Formatter\TapExample1\Tap: is most definitely pending # TODO todo: write pending example
```

Comment:
Can you clarify? The first TODO is a directive, everything after that is the actual message (if any) straight from phpspec.